textworld/challenges/spaceship/spaceship_game.py

```diff
@@ -0,0 +1,411 @@
+import argparse
+import os
+
+from os.path import join as pjoin
+from typing import Mapping, Optional
+
+import textworld
+from textworld import g_rng
+from textworld import GameMaker
+from textworld.challenges import register
+from textworld.generator.data import KnowledgeBase
+from textworld.generator.game import GameOptions, Event, Quest
+
+
+g_rng.set_seed(20190826)
+PATH = pjoin(os.path.dirname(__file__), 'textworld_data')
+
+
+def build_argparser(parser=None):
+    parser = parser or argparse.ArgumentParser()
+
+    group = parser.add_argument_group('Spaceship game settings')
+    group.add_argument("--level", required=True, choices=["easy", "balanced", "sparse"],
+                       help="The difficulty level. Must be between: easy, medium, or difficult.")
+
+    return parser
+
+
+def make_game(settings: Mapping[str, str], options: Optional[GameOptions] = None) -> textworld.Game:
+    """ Make a Spaceship game of the desired difficulty settings.
+
+    Arguments:
+        settings: Difficulty settings (see notes).
+        options:
+            For customizing the game generation (see
+            :py:class:`textworld.GameOptions <textworld.generator.game.GameOptions>`
+            for the list of available options).
+
+    Returns:
+        Generated game.
+
+    Notes:
+        Difficulty levels are defined as follows:
+        * Level easy.
+        * Level medium.
+        * Level difficult.
+
+    """
+    kb = KnowledgeBase.load(target_dir=PATH)
+    options = options or GameOptions()
+    options.grammar.theme = 'Spaceship'
+    options.kb = kb
+    options.seeds = g_rng.seed
+
+    rngs = options.rngs
+    rng_map = rngs['map']
+    rng_objects = rngs['objects']
+    rng_grammar = rngs['grammar']
+    rng_quest = rngs['quest']
+
+    if settings["level"] is 'easy':
```

Comment:
What you want to use for string comparisons is `==`  instead of `is`.

Reply:
This has already been done on the code.